tools/ui-components/src/image/image.tsx

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 import { ImageProps } from './image.types';
 
 export const Image = React.forwardRef<HTMLImageElement, ImageProps>(
-  ({ alt, src, responsive, ...props }, ref): JSX.Element => {
+  ({ alt, src, responsive, className, ...props }, ref): JSX.Element => {
```

Comment:
Do we want to set "responsive" as the default? What happens when the component does not specify the `className` or the `reponsive` prop?